### DISCUSSION TITLE
Is there a way to pass type = {\bibnotype} when item type = report?

### DISCUSSION THREAD

**Discussion by DS-Rodrigues** [retorquere/zotero-better-bibtex#2476] [Q&A] [answered]
Is there a way to pass type = {\bibnotype} in the bib file when item type = report?

If I just include {\bibnotype} in the Report type field in Zotero then Better BibTeX pass it as:

type = {\{\textbackslash bibnotype\}}

which does not seem to work for me (ie, remove (Tech. Rep.) from reference). Only if {\bibnotype} is stated in type field.

Many thanks!

**Comment by retorquere** [comment, marked as answer]
Add `tex.type = \bibnotype` to the `extra` field, or enter `<script>\bibnotype</script>` in the `type` field.

**Comment by DS-Rodrigues** [comment]
It works, thank you!
